lib/rexml/parsers/baseparser.rb

```diff
@@ -601,36 +602,36 @@ def parse_attributes(prefixes, curr_ns)
         return attributes, closed if raw_attributes.nil?
         return attributes, closed if raw_attributes.empty?
 
-        scanner = StringScanner.new(raw_attributes)
-        until scanner.eos?
-          if scanner.scan(/\s+/)
-            break if scanner.eos?
+        @attributes_scanner.string = raw_attributes
```

Comment:
How about reduce changes by introducing `scanner` local variable? (Shorter but meaningful variable will also keep the code base clean.)

```suggestion
        @attributes_scanner.string = raw_attributes
        scanner = @attributes_scanner
```

Reply:
OK, I see.
I will change it.